harvard-lts/CURIOSity
# Adjusting iframe wrapper height to 600px

**Adjusting iframe wrapper height to 600px*
* * *

# What does this Pull Request do?
This solves the issue of the iframe being too tall/large (since 100vh takes up entire screen) for urn items and removes the large blank space below the viewer for invalid urn items. 

# How should this be tested?

* Spin up curiosity and look at both valid and invalid urn items in Chrome, Safari, and Firefox; the height of the viewer should be 600px in every view/browser
* Example of invalid urn item: search for "Shen lun ; Quan shan liang yan..." in the demo site
* Example of valid urn item: any other item in demo site

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz @f8f8ff
